x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_event_log_list_table.tsx

```diff
@@ -0,0 +1,616 @@
+/*
```

Comment:
Should we have a unit test for this? This seems like the main component of this feature

Reply:
Resolved in this commit 38e4176c498707b99d0e22f55dac1349c2a98ad6